payment.js

```diff
@@ -0,0 +1 @@
+payment functino
```

Comment:
sai chính tả rồi e

Reply:
Em sua roi update PR

Reply:
Add a new comment to test the review feature